README.md

```diff
@@ -1,119 +1,189 @@
+facilitates classifier calibration testing, and finally allows to study the impact of the device 
+settings on the classification performance.
+
+![Species Similarities and differences](figures/fungi_samples.png)
+
+## News
+- We made a new train/test split based on ObservationIDs where data from the same observation do not occur in the test set. 
+- To distinguish it from the original split we call it **DanishFungi24**.
+- Updated baseline performance. All models are retrained and the results bellow updated (there is just a small drop in performance).
+- Model checkpoints are newly available at [Hugging Face Hub Repository](https://huggingface.co/BVRA).
+
+## Data
+| Subset                   | Images (full-size)                                                                        | Images <br/>(max side size 300px)                                                       | Metadata                                                           |
+|--------------------------|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|--------------------------------------------------------------------|
+| Danish Fungi 2020        | [LINK [~110GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-train_val.tar.gz) | [LINK [~6.5GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-train_val.tar.gz) | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20-metadata.zip) |
+| Danish Fungi 2020 – Mini | [LINK [~12.5GB]](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20M-images.tar.gz)  | ---                                                                                     | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DF20M-metadata.zip) |
+| Danish Fungi 2024        | ❗Same as for DF20❗                                                                        | ❗Same as for DF20❗                                                                      | [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DanishFungi2024.zip)                                                           |
+| Danish Fungi 2024 – Mini | ❗Same as for DF20 – Mini❗                                                                 | ---                                                                                     |    [LINK](http://ptak.felk.cvut.cz/plants/DanishFungiDataset/DanishFungi2024-Mini.zip)                                                   |
+
+
+To download the dataset files in CMD, use:
+```
+wget --no-check-certificate URL
+```
 
-Available at -> https://sites.google.com/view/danish-fungi-dataset
+In order to support research in fine-grained plant classification and to allow full reproducibility of our results, we share the training scripts and data tools.
+- Checkpoints are available at [Hugging Face Hub Repository](https://huggingface.co/BVRA).
```

Comment:
maybe it could link directly to https://huggingface.co/collections/BVRA/danish-fungi-2020-66a2228d0f4902df59d549e8